app/src/main/java/cs5150athletetracking/com/athletetracking/Uploader.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Uploader implements Runnable {
+	// URL should be updated. We are currently using a test script "test_server.php"
+	private static final String URL = "http://54.165.208.160/php/test_server.php";
+	private static final String TAG_SUCCESS = "success";
+	private String timestamp;
+	private String username;
+	private String bibnumber;
+	private String raceid;
+	private String latitude;
+	private String longitude;
+	private String altitude;
+
+	public Uploader(String user, int bib, int race, double lat, double lon, double alt) {
+		this.timestamp = Long.toString(System.currentTimeMillis());
+		this.username  = user;
+		this.bibnumber = Integer.toString(bib);
+		this.raceid    = Integer.toString(race);
+		this.latitude  = Double.toString(lat);
+		this.longitude = Double.toString(lon);
+		this.altitude  = Double.toString(alt);
+	}
+
+	@Override
+	public void run() {
+		System.out.println("Uploader Running...");
+		JSON jsonParser = new JSON();
+
+		// Building JSON Pairs...
+		List<NameValuePair> params = new ArrayList<NameValuePair>();
+		params.add(new BasicNameValuePair("timestamp", this.timestamp));
```

Comment:
I made a JSON object for this that we may want to use. We need to consider that sending up such long names will use a lot of data over the course of a few hours.